torch/sparse/__init__.py

```diff
@@ -91,6 +91,57 @@ def mm(mat1: Tensor, mat2: Tensor) -> Tensor:
     return torch._sparse_mm(mat1, mat2)
 
 
+sampled_addmm = _add_docstr(_sparse.sparse_sampled_addmm, r"""
+sparse.sampled_addmm(input, mat1, mat2, *, beta=1., alpha=1., out=None) -> Tensor
```

Comment:
Hey @IvanYashchuk! It looks like this op is in the public python API for the 1.11 release (`torch.sparse.sampled_addm`), but I don't think these docs made it into the website (I think you need to modify the docs .rst file for that?). Do you mind making a patch and cherry-picking it into the 1.11 release branch?

Reply:
Right, that was an oversight. I submitted a PR: https://github.com/pytorch/pytorch/pull/72312.